specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -192,11 +185,22 @@ function TreeView<SCHEMA extends AnyTree>({
         >
           {commonText.collapseAll()}
         </Button.Small>
+        <Button.Small
+          aria-pressed={isSplit}
+          onClick={() => setIsSplit(!isSplit)}
+        >
+          {treeText.splitView()}
+        </Button.Small>
```

Comment:
(optional) this is up to you and Grant and others, but you may want to consider having just one button for this, and have that button go between 3 states: no split view, horizontal split view, and vertical split view